README.md

```diff
@@ -151,6 +148,19 @@ fun SampleView() {
 
 As a convenience, if the retained instance implements the `OnClearedListener` interface, it will be automatically added to `onClearedListeners` and notified.
 
+#### View support
+
+Besides Activities and Fragments, it's also possible to retain instances in a view. There are a couple of extra modules for that:
+
+```gradle
+dependencies {
+    implementation 'com.github.marcellogalhardo.retained:retained-view:{Tag}'
+    implementation 'com.github.marcellogalhardo.retained:retained-view-navigation:{Tag}'
+}
+```
+
+The `retained-view` module exposes `retainInActivity` and `retain`, which will scope the instance to the parent being it an activity or a fragment. The `retained-view-navigation` module exposes `retainInNavGraph` to retain instances scoped to the `NavGraph`.
+
```

Comment:
I'm not sure if you're still planning to keep the view modules around, but if yes (I vote yes!), I think this would be a good way to talk about them, within the "Advanced usage" section.

There's no need to go into much details here since we can assume by now users understand how it works on activities and fragments, so the goal here is to just mention the extra view support and the necessary dependencies.

Reply:
Oh no  I think the correct name might be `retained-navigation-view` instead of `retained-view-navigation`.

Reply:
For now, I flag them as experimental and kept them around! I want to revisit their API and try to collect feedback of users to see if it is worth or not to maintain it. After all, with Compose hitting stable on July and Viewmodel not supporting view I'm not sure if we will have adoption here... 🤔